### ISSUE TITLE
Pages stop rendering when switching between pages rapidly.

### ISSUE THREAD

**Issue by cronco** [hrynko/vue-pdf-embed#194]
When switching pages rapidly, the canvas sometimes stops showing and we get the following error in the `rendering-failed` event listener:
```
Cannot use the same canvas during multiple render() operations. Use different canvas or ensure previous operations were cancelled or completed.
```
The stack trace traces it to `initializeGraphics` in pdf.js.

**Comment by hrynko** [comment]
Hi @cronco,

Could you provide more details about the environment as well as the version of `vue-pdf-embed` you are using?

**Comment by cronco** [comment]
Tested on Mac Sonoma on latest Chrome (122.0.6261.112 (Official Build) (arm64)), but have got reports of users on Chrome on Windows seeing the same issue.

The production app is on `vue-pdf-embed` 1.2.1, but I am testing 2.0.2 and I am getting similar results.

**Comment by hrynko** [comment]
A fix for this issue has been released in `v2.1.0`. Thanks for reporting!
